content/donate.fr.md

```diff
@@ -0,0 +1,58 @@
+{
+  "title": "Faire un don",
+  "type": "faire-un-don",
+  "aliases": ["spenden", "verein/donate", "verein/spenden"]
+}
+
+Merci beaucoup de ton soutien ! Ton don nous aidera à financer nos projets actuels et futurs afin qu'ils restent gratuits pour tous.
+
+Si tu souhaites nous soutenir de manière récurrente, tu peux également [devenir membre]({{< ref "become-a-member" >}}). De cette façon, tu peux également nous fournir un mandat de prélèvement SEPA.
+
+En tant qu'organisation d'utilité publique, nous pouvons également te délivrer un [reçu de don](#donation-receipt-howto).
```

Comment:
I am assuming that French tax authorities won't accept donations to us for tax deductions. As such, this sentence is probably more confusing than helping.

Let's just delete this sentence here and leave the explanation below.

Reply:
Actually I think you're wrong. "The body may be in France or a Member State of the **European Union**, Iceland, Norway or Liechtenstein" as written here https://www.service-public.fr/particuliers/vosdroits/F426 (French only)

Reply:
Oh nevermind. Good to know, thanks for checking! In that case, this sentence can of course stay.